native/CefBrowser_N.cpp

```diff
@@ -1902,11 +1926,11 @@ Java_org_cef_browser_CefBrowser_1N_N_1SendMouseWheelEvent(
   cef_event.x = x;
   cef_event.y = y;
 
-  cef_event.modifiers = GetCefModifiers(env, cls, modifiers);
+  cef_event.modifiers = GetCefModifiersGlfw(env, cls, modifiers);
 
-  if (scroll_type == JNI_STATIC(WHEEL_UNIT_SCROLL)) {
+  if (scroll_type == 0) {
```

Comment:
Add a comment saying this means WHEEL_UNIT_SCROLL

Reply:
Good point